### ISSUE TITLE
[BUG] Message suggesting a user induced reboot appears prematurely - rebooting breaks the install

### ISSUE THREAD

**Issue by axsms** [harvester/harvester#2142]
**Describe the bug**
At one point during the ISO install, the below message appears.
```
Done executing stage 'after-install.after'
Deployment done, now you might want to reboot
```
There are several actions/scripts remaining and rebooting at that point results in a broken install.

**To Reproduce**
1. ISO install via virtual media drive

**Expected behavior**
Install completes and all harvester features are available through the dashboard.

**Support bundle**
The install does not complete in order to get a support bundle in the manner described. Rancher installs, but not Harvester.

**Environment:**
 - Harvester ISO version: 1.0.1
 - Underlying Infrastructure: HP DL360 G9

**Additional context**
There is a long break in messaging after the above message appears, which reinforces the idea that we should reboot. I would have likely repeated the action had I not left the install running overnight.

**Comment by tlehman** [comment]
Blocked on VPN access to get to HP DL360p Gen9 from data center

**Comment by bk201** [comment]
The message is produced by the legacy `cos` script.
We are [migrating to `elemental`](https://github.com/harvester/harvester-installer/pull/273) and the tool should not output this message anymore.

**Comment by bk201** [comment]
https://github.com/harvester/harvester-installer/pull/273 was in. To verify this issue, just check the misleading message `Deployment done, now you might want to reboot` is gone during the installation.

**Comment by TachunLin** [comment]
Verified **fixed** on `v1.0.2-rc1`. Close this issue.

### Result
During ISO install Harvester, we did not see message related to 
```
Done executing stage 'after-install.after'
Deployment done, now you might want to reboot
```
And the ISO installation seems faster than before

![image](https://user-images.githubusercontent.com/29251855/165920911-474db30f-a2a5-44c7-88a2-51526f6d3c28.png)


### Test Information
* Test Environment: 1 node harvester on local kvm machine
* Harvester version: v1.0.2-rc1

### Verify Steps
1. ISO install Harvester
2. Monitor all message during the ISO installation process
